Src/BootCamp.Chapter/ConsoleLogger.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class ConsoleLogger : Illogger
```

Comment:
Messy interface name Illoger. Please remove the extra letters.

Reply:
done